lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -51,12 +51,12 @@ def check_config(self, client, server_config, host, my_headers={}):
                 "results": f"{host}/results",
                 "elasticsearch": f"{uri}/elasticsearch",
                 "endpoints": f"{uri}/endpoints",
-                "index_mappings": f"{uri}/index/mappings/",
-                "index_search": f"{uri}/index/search",
                 "graphql": f"{uri}/graphql",
                 "controllers_list": f"{uri}/controllers/list",
+                "dataset_mappings": f"{uri}/dataset/mappings/",
                 "controllers_months": f"{uri}/controllers/months",
                 "datasets_namespace": f"{uri}/datasets/namespace/",
+                "dataset_summary": f"{uri}/dataset/summary",
```

Comment:
It would be good if this whole list were ordered, as well, either by the keys or by the values.